Docs - Minor edits Applying Style Guide and Diataxis (#1123)
iotaledger/bee: documentation/docs/setup_a_node.mdx

To run a Bee node in a macOS system, you will need to install the following packages using the [brew](https://brew.sh/) package manager:

- [cmake](https://cmake.org/).
- [npm](https://www.npmjs.com/).

You can run the following command to install brew:

```shell
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

> After the installer finishes, you can use the brew to install the required packages by running:

Reviewer: I think it's either "brew" on its own, or "the brew package manager".

Author: 👌

### Download the Source

> Once you have installed all the required dependencies, you can start compiling the Bee Node. To do so, you can simply clone the source code by running the following command.

Reviewer: ```suggestion
Once you have installed all the required dependencies, you can start compiling the Bee Node. To get started, you should clone the source code by running the following command:
```

```shell
git clone https://github.com/iotaledger/bee.git --branch dev
```

> #### Compiling

Reviewer: ```suggestion
### Compiling
```

Reviewer: I think the headers on lines 174 and 225 should be an h4 (####) as they seem to be a subsection on compiling.
